src/extension.ts

```diff
@@ -12,10 +12,24 @@ function activate ( context: vscode.ExtensionContext ) {
   Utils.state = context.globalState;
   Utils.initCommands ( context );
 
+  context.subscriptions.push ( vscode.window.onDidChangeWindowState ( onDidChangeWindowState ) );
+
   return statusbar.init ();
 
 }
 
+function onDidChangeWindowState ( event ) {
+
+  if (event.focused && Utils.state.get ( 'didOpenInBrowser', false ) ) {
+    
+    vscode.commands.executeCommand ( 'githubNotificationsBell.refresh', true );
```

Comment:
Can you call the command directly instead of using `executeCommand`?

Reply:
Forgot that 😅